### PR TITLE
Enable posting user stats to Tuneme slack channel

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@miltontony @saeedmb8 ready for review.
I'm not sure adding them to settings/base.py is the best choice since it will fire stats for QA as well but it needs to be testable on somehow.
We need to decide if QAing this should hold up release of the stuff that's already in QA before we merge.

**Comment by KaitCrawford** [comment]
@miltontony @saeedmb8 ready for review

File: tuneme/settings/base.py [hunk 1]
```diff
@@ -239,3 +240,15 @@
 
 # Whether to use face/feature detection to improve image cropping - requires OpenCV  # noqa
 WAGTAILIMAGES_FEATURE_DETECTION_ENABLED = False
+
+SLACK_INCOMING_WEBHOOK_URL = 'https://hooks.slack.com/services/T0CJ9CT7W/\
```
Comment (miltontony) [inline tuneme/settings/base.py]: I think this needs to be secret and added in puppet..

**Comment by miltontony** [comment]
:+1: - small question around the webhook url

**Comment by KaitCrawford** [comment]
@miltontony just checking that removing the webhook url entirely was the write thing to do. Or do I need to leave the variable name in the settings and overwrite it in puppet?

**Comment by miltontony** [comment]
Removing it was fine
:+1:

**Comment by KaitCrawford** [comment]
@saeedmb8 testing your email

**Pull request merged**